Sources/Host/Linux/PTrace.cpp

```diff
@@ -255,19 +255,16 @@ ErrorCode PTrace::suspend(ProcessThreadId const &ptid) {
 
 ErrorCode PTrace::step(ProcessThreadId const &ptid, ProcessInfo const &pinfo,
                        int signal, Address const &address) {
+#if defined(ARCH_ARM)
+  DS2BUG("single stepping for ARM must be implemented in software");
+#endif
```

Comment:
Yeah I just looked at this, and PTrace::step appears not only on linux, but also in the POSIX header, that's gonna be too many ifdefs, so I don't think it's worth the pain.